src/style/app.module.css

```diff
@@ -416,8 +420,9 @@
 
 .dropdownItem {
   background-color: var(--bs-white) !important;
-  color: var(--brown-color) !important;
+  color: var(--black-color) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`